<launch>
  <arg name="color_image_topic" default="/camera/color/image_raw"/>
  <arg name="depth_image_topic" default="/camera/depth/image_raw"/>
  <arg name="is_record_topic" default="/migrave_data_recording/is_record"/>
  <arg name="video_type" default="mp4"/>
  <arg name="video_dimensions_color" default="480p"/>
  <arg name="video_dimensions_depth" default="480p"/>
  <arg name="frames_per_second" default="30"/>
  <arg name="output_directory" default="/home/qtrobot/Documents"/>

  <node name="migrave_video_recorder" pkg="migrave_video_recorder" type="migrave_video_recorder" output="screen" respawn="true">
    <param name="color_image_topic" value="$(arg color_image_topic)" />
    <param name="depth_image_topic" value="$(arg depth_image_topic)" />
    <param name="is_record_topic" value="$(arg is_record_topic)" />
    <param name="video_type" value="$(arg video_type)" />
    <param name="video_dimensions_color" value="$(arg video_dimensions_color)" />
    <param name="frames_per_second" value="$(arg frames_per_second)" />
    <param name="output_directory" value="$(arg output_directory)" />
  </node>

  <node name="migrave_webcam_recorder" pkg="migrave_video_recorder" type="migrave_webcam_recorder" output="screen" respawn="true">
  </node>
  
</launch>
